src/i18n/nl/index.ts

```diff
@@ -87,89 +87,89 @@ const nl = {
   },
   SETTINGS: {
     THEME: {
-      SYSTEM: '',
-      LIGHT: '',
-      DARK: '',
+      SYSTEM: 'Systeem',
+      LIGHT: 'Licht',
+      DARK: 'Donker',
     },
     PASSWORD: {
       POLICY: {
-        TITLE: '',
-        UPPERCASE_LETTER: '',
-        LOWERCASE_LETTER: '',
-        NUMBER: '',
-        CHARACTERS: '',
+        TITLE: 'Wachtwoord',
+        UPPERCASE_LETTER: 'Hoofdletters',
+        LOWERCASE_LETTER: 'Kleine letters',
+        NUMBER: 'Nummers',
+        CHARACTERS: 'Karakters',
       },
     },
     RESET_APP: {
-      TITLE: '',
-      DESCRIPTION: '',
-      CONFIRM: '',
-      CANCEL: '',
+      TITLE: 'Reset app',
+      DESCRIPTION: 'Weet je zeker dat je de app wilt resetten en alle gegevens wilt verwijderen? ',
+      CONFIRM: 'Ja, reset alles',
+      CANCEL: 'Nee, bewaar mijn profiel',
     },
-    ACCOUNT: '',
+    ACCOUNT: 'Account',
     SUPPORT: {
-      TITLE: '',
+      TITLE: 'Support',
       ABOUT: {
-        TITLE: '',
+        TITLE: 'Over UniMe',
       },
       FEEDBACK: {
-        TITLE: '',
+        TITLE: 'Stuur feedback',
       },
     },
-    NAVBAR_TITLE: '',
+    NAVBAR_TITLE: 'Instellingen',
     PROFILE: {
-      TITLE: '',
+      TITLE: 'Mijn profiel',
       PROFILE_NAME: {
-        TITLE: '',
-        NAVBAR_TITLE: '',
-        INPUT_PLACEHOLDER: '',
-        CONFIRM: '',
+        TITLE: 'Profielnaam',
+        NAVBAR_TITLE: 'Bewerk profielfoto',
+        INPUT_PLACEHOLDER: 'Profielnaam invoeren',
+        CONFIRM: 'Updaten',
       },
       DISPLAY_PICTURE: {
-        TITLE: '',
-        EDIT: '',
-        CHANGE: '',
+        TITLE: 'Toon afbeelding',
+        EDIT: 'Bewerk',
+        CHANGE: 'Updaten',
```

Comment:
Opslaan

Reply:
I don't think "Opslaan" fits here. 

Original English version is "Select a profile picture", so a better translation imo would be something like:
* "Selecteer"
* "Selecteer een nieuwe afbeelding"

![image](https://github.com/impierce/identity-wallet/assets/50104940/78d3e841-5d66-42e1-ba67-1b8272c27a40)